independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json

```diff
@@ -0,0 +1,514 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "De Lijn Zoek Open Data Services (v1)",
+    "description": "Get locations of public bus stations and extra info from the Flemish bus company 'De Lijn'",
+    "version": "v1"
+  },
+  "host": "api.delijn.be",
+  "basePath": "/DLZoekOpenData/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/zoek/haltes/{zoekArgument}": {
+      "get": {
+        "summary": "zoekHaltes",
+        "description": "Search stops by description. Eg Veemarkt or station.",
+        "operationId": "ZoekHaltes",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "zoekArgument",
+            "description": "het zoekargument",
```

Comment:
@Lschockaert all parameters descriptions should be free of grammatical errors and typos. please see {here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters)
Please change this for all the parameters descriptions in the swagger file.